internal/transactionsubmission/README.md

```diff
@@ -1,10 +1,24 @@
 # Transaction Submission Service
 
-The Transaction Submission Service (TSS) is a component that is responsible for submitting payment transactions to the Stellar Network.
+The Transaction Submission Service (TSS) allows for transaction submission at scale to the Stellar network from a single source/distribution account, which is made possible through the use of [channel accounts](https://developers.stellar.org/docs/encyclopedia/channel-accounts) spawned from said distribution account. In the context of SDP, the TSS is responsible for submitting transactions to the network for each respective payment read from a shared database table that is "queued" by the SDP. It works seamlessly by way of graceful error handling and transaction retries alongside mechanisms such as transaction fee bumps and adjustment of the rate at which new transactions are polled when certain errors are encountered.
```

Comment:
If a person who doesn't know much about the Stellar network were to read the term `transaction fee bumps`, they would probably think the TSS changes the gas fees dynamically, which is not the case.

I suggest mentioning "sponsored reserves" and "fee bump transactions" in a separate paragraph with links to the docs explaining what it's about.